node_modules/oae-tenants/tests/test-tenants.js

```diff
@@ -445,6 +445,28 @@ describe('Tenants', function() {
         });
 
         /**
+         * Test that verifies that creating a tenant with an alias that contains a dash does not
+         * break the authentication strategy data of the `me` feed. This is a regression test for
+         * https://github.com/oaeproject/Hilary/issues/1172
+         */
+        it('verify creating a tenant with a dash in the alias does not break authentication strategy', function(callback) {
+            var tenantHost = TestsUtil.generateRandomText(1);
+            var tenantAlias = 'test-with-dash';
+            TestsUtil.createTenantWithAdmin('alias-with-dash', tenantHost, function(err, tenant, tenantAdminRestContext) {
+                assert.ok(!err);
+
+                // Ensure the tenant admin's me object propertly represents the authentication
+                // strategy
+                RestAPI.User.getMe(tenantAdminRestContext, function(err, me) {
```

Comment:
@mrvisser : I know this is almost definitely the same, but would you mind verifying this with a regular user instead of an admin user?